src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -371,9 +372,11 @@
     <Reference Include="System.Runtime.InteropServices.RuntimeInformation" />
     <Reference Include="System.ComponentModel" />
     <Reference Include="System.ComponentModel.Primitives" />
+    <Reference Include="System.ComponentModel.TypeConverter" />
```

Comment:
Thanks Santi I missed both those things

Reply:
Put them to `System.Windows.Extensions` like all other similar converters?

Reply:
Yes and use the full type name in the attribute to make it a runtime dependency rather than a compile dependency because S.W.Extensions depends on TypeConverters and if we depend on it, it will end up pulling it into the closure.

Just like we did in the PR that I linked.